tests/api_test.py

```diff
@@ -63,7 +63,7 @@ def jit(self):
     # TODO(jblespiau,phawkins): Remove this when jaxlib has been released.
     # This is in the future, because we are making a breaking change to
     # Tensorflow.
-    if version < (0, 1, 54):
```

Comment:
Nit: shouldn't we just skip the test? Won't `PythonJitTest` also run the Python tests?

Reply:
it's not just the test we need to skip, it's all the tests of this class.

It's a base class used by PythonJitTest, so if disable tests, it will also be disabled there.

By doing this, it will run CPPJitTest, but as the PythonTests

Reply:
Doesn't PythonJitTest override this method?